components/pkg-export-kubernetes/src/topology.rs

```diff
@@ -37,6 +59,16 @@ impl FromStr for Topology {
 }
 
 impl fmt::Display for Topology {
+    /// Basicially it converts a Topology into a string
```

Comment:
I'd write "Converts a Topology into a string", without the "Basically" part (with a typo :P).

Reply:
Agreed. If this is needs to be a documentation comment, it should sound like one.